osprey_flask_app/run_rvic.py

```diff
@@ -2,8 +2,7 @@
 from wps_tools.testing import get_target_url, url_path
 
 
-def run_full_rvic(arg_dict):
-    url = get_target_url("osprey")
+def run_full_rvic(arg_dict, url=get_target_url("osprey")):
```

Comment:
Consider making the `url` a parameter for this method and just use `get_target_url` as a default.

Reply:
Is that not what I do here?

Reply:
Yes, sorry I misread the code :+1: